applications/flash_attention_v2/collective/xe_flash_attn_mma.hpp

```diff
@@ -110,27 +109,16 @@ struct CollectiveMmaAttention<MainloopIntelPVC<Stages>, TileShape_, ElementQ_, S
 
   using SubgroupTileShapePV = decltype(cute::shape_div(TileShapePV{}, take<1, 4>(typename TiledMmaQVO::ThrLayoutVMNK{}.shape())));
 
-  static constexpr auto PV_SG_M = get<0>(SubgroupTileShapePV{});
-  static constexpr auto PV_SG_N = get<1>(SubgroupTileShapePV{});
-  static constexpr auto PV_SG_K = get<2>(SubgroupTileShapePV{});
-
   static constexpr auto QK_BLK_M = get<0>(TileShapeQK{});
   static constexpr auto QK_BLK_N = get<1>(TileShapeQK{});
   static constexpr auto QK_BLK_K = get<2>(TileShapeQK{});
 
-  using K_NUM_SG = decltype(typename TiledMmaQVO::ThrLayoutVMNK{}.shape());
   using K_Atom_Shape = decltype(Shape<Int<PV_ATOM_M>, _1, _1>{});
-  using K_Tile_Shape = decltype(Shape<_8, Int<PV_ATOM_M>, _2>{});
 
   // This TiledMma is only required to serve the specific tiling requirements for matrix K.
-  // TODO (Tanvir): Replace this instantiation with ContigBlockMMAHelper once available
-  using TiledMmaK = TiledMMA<typename TiledMmaQVO::Atom, Layout<K_Atom_Shape, Stride<_1, _1, _1>>,
-                            // Atom, Hardware(NUMBER OF CONCURRENT MMA), Iteration
-                            Tile<Layout<K_Tile_Shape, Stride<_1, _16, _8>>,   // Vec Iteration, Hardware Jump,
-                                                                                    // Iteration Jump for both M and N
-                                  Layout<Shape<_16, _1, _4>, Stride<_1, _64, _16>>, // Vec Iteration, Hardware Jump,
-                                                                                    // Iteration Jump for both M and N
-                                  _64>>;                                            // K is going to be 64
+  using TiledMmaK = typename TiledMMAHelper<typename TiledMmaQVO::Atom, 
+                                            Layout<TileShapeQK>,
+                                            Layout<K_Atom_Shape, Stride<_1, _1, _1>>>::TiledMMA;
```

Comment:
Does the stride always 1,1,1? I think the stride should be ready from the config pass py user

Reply:
This stride always needs to be `<1, 1, 1>` because we don't tile the `K` matrix. Given the current code, I think we can also replace the `K_Atom_Shape` with `Shape<_1, _1, _1>` because we don't use `mode<0>(K_Atom_Shape)` anywhere in the code.